gulpfile.js

```diff
@@ -104,7 +125,7 @@ const generateIconNames = (done) => {
 };
 
 gulp.task('generate-icon-names', generateIconNames);
-gulp.task('template', gulp.series('generate-icon-names', generateTemplates(parseTokens(tokens))));
+gulp.task('template', gulp.series('generate-icon-names', generateTemplates(parseUIKitTokens(tokens)), generateSwiftUI(parseSwiftUITokens(tokens))));
```

Comment:
Do we want to rename `generateTemplates` to something else maybe UIKit or ObjC related naming to avoid confusion and future maintenance?

Reply:
good catch!